Makefile

```diff
@@ -214,65 +220,61 @@ require-gh-token:
 	@if [[ -z "$$GH_TOKEN" ]]; then echo "GH_TOKEN environment value is required."; exit 1; fi
 
 require-secrets-dir:
-	if [[ -z "$$SECRETS_DIR" ]]; then echo "SECRETS_DIR environment value is required."; exit 1; fi
+	@if [[ -z "$$SECRETS_DIR" ]]; then echo "SECRETS_DIR environment value is required."; exit 1; fi
 
 require-secrets: require-secrets-dir
 	@if [[ -z "$$BNS_CERT_PASS" ]]; then echo "BNS_CERT_PASS environment value is required."; exit 1; fi
 
+$(BNS_CERT): require-secrets-dir
+	@cp -f $$SECRETS_DIR/$(BNS_CERT) $(BNS_CERT)
+
 require-lantern-binaries:
 	@if [[ ! -d "$(LANTERN_BINARIES_PATH)" ]]; then \
 		echo "Missing lantern binaries repository (https://github.com/getlantern/lantern-binaries). Set it with LANTERN_BINARIES_PATH=\"/path/to/repository\" make ..." && \
 		exit 1; \
 	fi
 
-docker-package-debian-386: require-version docker-linux-386
+package-linux-386: require-version linux-386
 	@cp lantern_linux_386 lantern_linux_i386;
 	@$(call fpm-debian-build,"i386")
 	@rm lantern_linux_i386;
 	@mv update_linux_i386.bz2 update_linux_386.bz2 && \
 	echo "-> lantern_$(VERSION)_i386.deb"
 
-docker-package-debian-amd64: require-version docker-linux-amd64
+package-linux-amd64: require-version linux-amd64
 	@$(call fpm-debian-build,"amd64")
 	@echo "-> lantern_$(VERSION)_amd64.deb"
 
-docker-package-debian-arm: require-version docker-linux-arm
+package-linux-arm: require-version linux-arm
 	@$(call fpm-debian-build,"arm")
 	@echo "-> lantern_$(VERSION)_arm.deb"
 
-docker-package-windows: require-version docker-windows-386
-	@if [[ -z "$$BNS_CERT" ]]; then echo "BNS_CERT environment value is required."; exit 1; fi && \
-	if [[ -z "$$BNS_CERT_PASS" ]]; then echo "BNS_CERT_PASS environment value is required."; exit 1; fi && \
+package-windows: $(BNS_CERT) do-package-windows
+
+do-package-windows: require-version windows
```

Comment:
No, because package-windows can't be run inside docker because the $(BNS_CERT) target doesn't work there (since the $SECRETS_DIR is not accessible from docker).